src/server.js

```diff
@@ -1,13 +1,46 @@
-const express = require("express")
-const morgan = require("morgan")
-const cors = require("cors")
-const app = express()
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const app = express();
+let contacts = require("../data/contacts.js");
+let meetings = require("../data/meetings.js");
 
-app.use(morgan("dev"))
-app.use(cors())
-app.use(express.json())
+app.use(morgan("dev"));
+app.use(cors());
+app.use(express.json());
 
 // write your app code here
 
+app.get("/contacts", function (req, res) {
+  res.status(200).json({ contacts: contacts });
+});
 
-module.exports = app
+app.post("/contacts", function (req, res) {
+  const postContact = req.body;
+  const currentHighId = contacts.reduce((max, obj) => {
```

Comment:
nice, you could make this a generic function in a utils.js file :

```js
function getNextId(items) {
   return items.reduce(...) + 1
}
```

and then call your function when needed, makes the code tidier